pom.xml

```diff
@@ -12,7 +13,10 @@
     <description>Allows Jenkins administrators to control what in-process scripts can be run by less-privileged users.</description>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Script+Security+Plugin</url> 
     <properties>
-      <findbugs-maven-plugin.version>3.0.2</findbugs-maven-plugin.version>
+      <jenkins.version>1.509.4</jenkins.version>
```

Comment:
This is too old. I just checked and found that the oldest baseline used by a dependency of this plugin is now 1.580.1, so I might as well update to that here too. Will do so in a separate PR.